spec/lib/vmdb/settings_spec.rb

```diff
@@ -178,6 +178,37 @@
       change = miq_server.settings_changes.find_by(:key => "/authentication/user_proxies")
       expect(change.value).to eq [{:bind_pwd => encrypted}]
     end
+
+    it "saving settings for Zone does not change saved Region or Server settings" do
+      MiqRegion.seed
+
+      described_class.save!(miq_server.zone, :api => {:token_ttl => "2.hour"})
+      miq_server.zone.reload
+      expect(miq_server.zone.settings_changes.count).to eq 1
+      expect(miq_server.zone.settings_changes.first).to have_attributes(:key   => "/api/token_ttl",
+                                                                        :value => "2.hour")
+      miq_server.reload
+      expect(miq_server.settings_changes.count).to eq 0
+
+      miq_server.zone.miq_region.reload
+      expect(miq_server.zone.miq_region.settings_changes.count).to eq 0
+    end
+
+    it "saving settings for Region does not change saved Zone or Server settings" do
+      MiqRegion.seed
+
+      described_class.save!(miq_server.zone.miq_region, :api => {:token_ttl => "3.hour"})
+      miq_server.zone.miq_region.reload
+
+      expect(miq_server.zone.miq_region.settings_changes.count).to eq 1
+      expect(miq_server.zone.miq_region.settings_changes.first).to have_attributes(:key   => "/api/token_ttl",
```

Comment:
Instead of calling `miq_server.zone.miq_region` in all these places, can you call `miq_server.miq_region` directly?  I find it confusing with both layers there.